package.json

```diff
@@ -4,8 +4,8 @@
   "main": "lib/index.js",
   "description": "Lightweight update server for Electron apps",
   "scripts": {
-    "dev": "npx vercel dev",
```

Comment:
Newer versions of the vercel CLI don't allow you to recursively call `dev` from `dev`, so I renamed this to `local`